build-jack2.sh

```diff
@@ -126,26 +126,17 @@ fi
 if [ -f "${PAWPAW_PREFIX}/bin/moc" ]; then
     download qjackctl "${QJACKCTL_VERSION}" https://download.sourceforge.net/qjackctl
 
-    if [ "${WIN64}" -eq 1 ]; then
-        patch_file qjackctl "${QJACKCTL_VERSION}" "configure" 's/-ljack /-Wl,-Bdynamic -ljack64 -Wl,-Bstatic /'
-    elif [ "${WIN32}" -eq 1 ]; then
-        patch_file qjackctl "${QJACKCTL_VERSION}" "configure" 's/-ljack /-Wl,-Bdynamic -ljack -Wl,-Bstatic /'
-    fi
-
-    if [ "${MACOS}" -eq 1 ]; then
-        qjackctl_extra_args="--with-jack="${jack2_prefix}${jack2_extra_prefix}""
-    elif [ "${WIN32}" -eq 1 ]; then
-        qjackctl_extra_args="--enable-portaudio"
-    fi
-
-    if [ "${MACOS_UNIVERSAL}" -eq 1 ]; then
-        export EXTRA_CXXFLAGS="-std=gnu++11"
+    if [ "${WIN32}" -eq 1 ] || [ "${LINUX}" -eq 1 ]; then
+        # Use system's Qt tools for translations
+        qjackctl_CMAKE_PREFIX_PATH="${PAWPAW_PREFIX}/lib/cmake;/usr/lib/x86_64-linux-gnu/cmake"
```

Comment:
Hardcoding `/usr/lib/x86_64-linux-gnu/cmake` is very ugly, not to mention this will easily break, ArchLinux for example doesn't use this path.
Can we call and ask cmake for this path/variable?

Reply:
I replied on the new diff sorry didnt see this one first.
Will mark this one as resolved. Also, let me have a try and see if I can find something...